plugins/available/pip.plugin.bash

```diff
@@ -0,0 +1,23 @@
+pip-upgrade-all() {
```

Comment:
please add `cite about-plugin` and `about-plugin` values, and add `about` and such for each function (see `fzf` plugin for reference)